index.js

```diff
@@ -1 +1,49 @@
-console.log("Hello World!")
\ No newline at end of file
+const axios = require('axios');
+
+const NASA_ASTEROIDS_FEED_URL = 'https://api.nasa.gov/neo/rest/v1/feed';
+const API_KEY = 'eLmnePyuHtTVKYHiKBC7fVH4weBUtTMiZIGoZftC';
```

Comment:
could you add dotenv package and move this token to .env file?
storing sensitive data in code is not a good practice

A couple of links that explain the reasons in more detail
https://stackoverflow.com/questions/63319800/why-do-people-use-env-file-on-server
https://www.npmjs.com/package/dotenv